packages/scss/src/components/inputs/switch/_switch.scss

```diff
@@ -108,3 +80,48 @@
 		}
 	}
 }
+
+
+// PALETTE
+// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+.switch {
+	@mixin switchColoring($palette) {
+		.switch-input:checked {
+			~ .switch-label {
+				&::before {
+					background: lighten(_get($palette, "color"), 20%);
+				}
+
+				&::after {
+					background: _get($palette, "color");
+				}
+			}
+
+			// Palette > Disabled switch 
+			&[disabled], &[readonly]{
+				&:checked {
```

Comment:
You're already selecting `.switch-input:checked`, this one doesn't seem necessary